save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ContestView.kt

```diff
@@ -58,7 +70,14 @@ external interface ContestViewState : State {
 @OptIn(ExperimentalJsExport::class)
 class ContestView : AbstractView<ContestViewProps, ContestViewState>(false) {
     init {
-        state.selectedMenu = ContestMenuBar.INFO
+        state.selectedMenu = ContestMenuBar.defaultTab
+    }
+
+    override fun componentDidMount() {
+        super.componentDidMount()
+
+        ContestMenuBar.setPath("#/${FrontendRoutes.CONTESTS.path}/${props.currentContestName}", "#/contests/${FrontendRoutes.CONTESTS.path}/${props.currentContestName}")
```

Comment:
Why are you setting it here and not in `ContestMenuBar` definition?

Reply:
```Props.currentContestName``` is stored there in the address, how can I specify this in advance?

Reply:
Ok